index.html

```diff
@@ -2,12 +2,77 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
 
+    <div class="wrapper">
+       
+        <div class="container">
+            <div  class="hour">      9:31 </div>
+            <div  class="indicators"> 
+                <img src="./images/status_mobile.png">
+            </div>
+
+        </div>
```

Comment:
I think you did a great job implementing this block of code. However, this is not part of the project. Kindly remove it. :recycle: :recycle: